app/src/main/res/values/strings.xml

```diff
@@ -113,12 +113,12 @@
     <!-- Sample Text -->
     <string name="sample_text_station_metadata" translatable="false">David Bowie - The Man Who Sold The World</string>
     <string name="sample_text_station_name" translatable="false">Byte FM</string>
-    <string name="sample_text_steam_url" translatable="false">http://bytefm.cast.addradio.de/bytefm/main/mid/stream</string>
+    <string name="sample_text_steam_url" translatable="false">https://bytefm.cast.addradio.de/bytefm/main/mid/stream</string>
     <string name="sample_text_sleep_timer_remaining_time" translatable="false">00:00</string>
-    <!-- Allowed Caller Template -->
-    <string name="allowed_caller_log" translatable="false">
-        Caller has a valid certificate, but its package doesn\'t match any expected package for the
-        given certificate. To allow this caller, add the following to the allowed callers list:\n
+    <!-- Allowed Template -->
+    <string name="allowed_caller_log">
+        The caller has a valid certificate, but the wrong package for it.
+        To allow this caller, add the following to the allowed callers list:\n
```

Comment:
Where is the "Allowed callers" list?

Reply:
I removed the whole string, since it causes errors, as you pointed out a while a ago.

See: https://github.com/y20k/transistor/commit/aa9c7a8ce86cb03af38b079d4e7c29f7660f224f

Reply:
I removed it. Could be uncommenting it is better. It caused problems with XML checks over at Weblate.